modules/apps/site/site-memberships-web/src/main/resources/META-INF/resources/js/actions.js

```diff
@@ -28,12 +29,21 @@ export const ACTIONS = {
 					return;
 				}
 
-				const input = document.createElement('input');
+				const allItems = document.createElement('input');
```

Comment:
Variable name is too generic, you're creating a single `input` element, but naming it `allItems`.

Reply:
I would have thought `input` would be more generic, whereas `allItems` is referring to all the checkbox items...

Reply:
Well, first of all, you're just creating one input element here, right? So the name `allItems` makes no sense.

Even if you were storing all checkbox items in the variable, a better name would be `checkboxItems` or `allCheckboxItems`.